README.md

```
@@ -180,6 +180,35 @@ proxy.execute(mode, abi.encode(calls, opData));
+- The SCA can upgrade the contract owner.
+
+The only way to have the SCA call itself is through the batch endpoint. The batch endpoint requires a structured signed message but the encoded messages themselves are not structured. It is very important to parse and validate that **ALL** signed batches are legit and safe. Take the following batch:
+
+- Call self, transferOwnership to A.
+- Call A
+- Call self, transferOwnership to original owner.
+
+If A implement ERC-1271, it can execute **ANY** call it desires on the SCA. Using the above batch, it is possible to:
+
+1. Call A with batch + custom calldata.
+2. Store custom calldata in transient storage and set custom calldata as signed	(by A).
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 80

(MD010, no-hard-tabs)